infrastructure/cdn-in-a-box/to_integration/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM golang:1.11 AS integration-builder
+
+# integration source and dependencies
+COPY ./lib/ /go/src/github.com/apache/trafficcontrol/lib/
```

Comment:
Can all these `COPY` lines be collapsed into single `COPY ./ /go/src/github.com/apache/trafficcontrol/`? Or if only those specific files/dirs are required, list them all in one `COPY` command?

Reply:
Yeah, it used to be best practice to reduce the total number of layers in your containers to keep their size down, but it seems like they've fixed that issue with multistage builds now (multiple FROMs). With the multistage build here I'm guessing it's better to leave your multiple COPYs as-is.